docs/local-custom-files.md

```diff
@@ -0,0 +1,48 @@
+# Build KubeVirt using local files
+
+KubeVirt system strongly relies on bazel and it is built mostly by files hosted remotely. In certain situations, it is handy to be able to build KubeVirt using custom local files. 
```

Comment:
Maybe listing some of those "certain" situations would be helpful. Consider adding a couple of the most common reasons.